src/tabletoolbar.js

```diff
@@ -8,17 +8,20 @@
  */
 
 import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
-import { isTableContentSelected } from './utils';
+import { isTableContentSelected, isTableWidgetSelected } from './utils';
 import WidgetToolbarRepository from '@ckeditor/ckeditor5-widget/src/widgettoolbarrepository';
 
 /**
- * The table toolbar class. It creates a table toolbar that shows up when the table content is selected.
+ * The table toolbar class. It creates toolbars for the table feature and its content (for now only for a table cell).
  *
- * Instances of toolbar components (e.g. buttons) are created using the editor's
- * {@link module:ui/componentfactory~ComponentFactory component factory}
- * based on the {@link module:table/table~TableConfig#toolbar `table.toolbar` configuration option}.
+ * Table toolbar shows up when the table widget is selected. Its components (e.g. buttons) are created based on the
+ * {@link module:table/table~TableConfig#toolbar `table.tableToolbar` configuration option}.
  *
- * The toolbar uses the {@link module:ui/panel/balloon/contextualballoon~ContextualBalloon} plugin.
+ * Table content toolbar shows up when the table cell is selected. It creates its component based on the
```

Comment:
I do not like "table cell is selected" part. We do not have any cell selection handling yet. The content toolbar shows up when the selection is in the content of the table. I think it will be simpler to describe it this way.